pages/projects_page.py

```diff
@@ -80,6 +82,34 @@ def rename_workspace(self, old_name, new_name):
                 self.click(self._CONFIRMATION_BUTTON)
                 break
 
+    @allure.step("Start a new project")
+    def create_new_project(self):
+        if self.is_elem_displayed(self._driver.find_element(*self._START_BUTTON)):
+            self.click(self._START_BUTTON)
+        elif self.is_elem_displayed(self._driver.find_element(*self._CREATE_PROJECT_BUTTON)):
+            self.click(self._CREATE_NEW_WORKSPACE_BUTTON)
+
+    @allure.step("Search for project {project_name}")
+    def search_project(self, project_name):
+        self.click(self._SEARCH_BUTTON)
+        self.fill_text(self._SEARCH_FIELD, project_name)
+
+    @allure.step("Delete or cancel deletion of project {project_name}")
+    def delete_project(self, project_name, status="confirm"):
```

Comment:
why do we pass hardcoded status in the signature and not in the test?

Reply:
This is a pythonic way to give an argument a default value.
https://www.tutorialspoint.com/What-are-default-arguments-in-python#:~:text=Python%20has%20a%20different%20way,by%20using%20assignment%20(%3D)%20operator.

Reply:
This is a pythonic way for giving an argument a default value:
https://www.tutorialspoint.com/What-are-default-arguments-in-python#:~:text=Python%20has%20a%20different%20way,by%20using%20assignment%20(%3D)%20operator.